src/ResourceGraph/ResourceGraph/help/Search-AzGraph.md

```diff
@@ -55,6 +55,18 @@ westus            26
 
 A complex query on resources featuring field selection, filtering and summarizing.
 
+### Example 3
+```powershell
+PS C:\> Search-AzGraph -Query 'where type =~ "Microsoft.Compute/virtualMachines"| where properties.storageProfile.osDisk.managedDisk == "" | project name, resourceGroup, subscriptionDisplayName' -Include DisplayName
+
+name         resourceGroup      subscriptionDisplayName
+----         -------------      -----------------------
+ContosoVM    RG-Contoso         Contoso Production Subscription                                               
+
+```
+
+A query featuring all virtual machines which are not using Managed Disks and includes subscription displayname.
+
```

Comment:
Thanks, @MattiasAng for improving the help. I appreciate your contribution.
Suggest adding example which includes tenantDisplayName also something like 
Search-AzGraph -Include DisplayName -Query 'summarize count() by tenantDisplayName, subscriptionDisplayName'

Reply:
Thanks. Added.